src/parse.c

```diff
@@ -4440,7 +4440,7 @@ void Parser::checkCstyleTypeSyntax(Loc loc, Type *t, int alt, Identifier *ident)
 
 Statement *Parser::parseStatement(int flags, const utf8_t** endPtr, Loc *pEndloc)
 {
-    Statement *s;
+    Statement *s = NULL;
```

Comment:
> Is this what you would expect to happen?

Update: instead of `goto Lerror`, I just set `s` to `NULL`, and it seems to work, will continue testing.